Source/Core/Core/HotkeyManager.cpp

```diff
@@ -109,6 +109,8 @@ const std::string hotkey_labels[] =
 	_trans("Load State Last 6"),
 	_trans("Load State Last 7"),
 	_trans("Load State Last 8"),
+	_trans("Load State Last 9"),
```

Comment:
@Armada651 I think only this two lines should go to stable. I'll create a new PR for stable soon.